middlewares/userValidations.js

```diff
@@ -263,5 +263,18 @@ export default {
     });
     inputTypeValidator(isString, inputArray, next);
     return next();
+  },
+
+  async isUsernameExists(req, res, next) {
```

Comment:
Please rename this variable to either `isUsernameExisting` or `doesUsernameExist`